src/components/Configuration/AppInfoModal.vue

```diff
@@ -30,7 +30,7 @@
       <!-- License -->
       <h3>{{ $t('app-info.license') }}</h3>
       {{ $t('app-info.license-under') }} <a href="https://github.com/Lissy93/dashy/blob/master/LICENSE">MIT X11</a>.
-      Copyright <a href="https://aliciasykes.com">Alicia Sykes</a> © 2021.<br>
+      Copyright <a href="https://aliciasykes.com">Alicia Sykes</a> © 2024.<br>
```

Comment:
In the this file and `src/views/About.vue`, we might be able to make this dynamic with:
`{{new Date().getFullYear()}}`

Reply:
Sounds great! I'll try to implement that!

Reply:
@Lissy93 Done and commited!